sys/include/net/ipv6/ext/opt.h

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright (C) 2020 Freie Universität Berlin
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    net_ipv6_ext_opt IPv6 destination and hop-by-hop options
+ * @ingroup     net_ipv6_ext
+ * @brief       Definitions for IPv6 destination and hop-by-hop extension
+ *              options extension headers
```

Comment:
```suggestion
 *              options headers
```

Reply:
They are extension headers that provide options, so I'll remove the first "extension" to result in 

```
 * @brief       Definitions for IPv6 destination and hop-by-hop options
 *              extension headers
```